velox/dwio/parquet/tests/ParquetTableScanTest.cpp

```diff
@@ -25,64 +25,279 @@ using namespace facebook::velox;
 using namespace facebook::velox::exec;
 using namespace facebook::velox::exec::test;
 
-static const std::string kWriter = "ParquetTableScanTest.Writer";
-
 class ParquetTableScanTest : public HiveConnectorTestBase {
  protected:
   using OperatorTestBase::assertQuery;
 
   void SetUp() override {
     HiveConnectorTestBase::SetUp();
     parquet::registerParquetReaderFactory();
+
+    loadData();
+  }
+
+  void assertSelect(
+      const std::vector<std::string>&& outputColNames,
+      const std::string& sql) {
+    std::vector<std::shared_ptr<const Type>> types;
+    for (auto colName : outputColNames) {
+      types.push_back(rowTypes->findChild(colName));
+    }
+
+    auto plan =
+        PlanBuilder()
+            .tableScan(
+                ROW(const_cast<::std::vector<std::string>&&>(outputColNames),
```

Comment:
Change the signature to take outputColumnNames by value and remove const_cast here for readability.

```
ROW(std::move(outputColNames), std::move(types))
```